app/middleware/cors.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('kcors');
+module.exports = require('@koa/cors');
```

Comment:
kcors rename to @koa/cors now

Reply:
will others koa official lib rename?

Reply:
I don't know, just because kcors is to strange